source/_integrations/plugwise.markdown

```
@@ -30,19 +30,30 @@ ha_platforms:
+- A stand-alone smart thermostat called [Anna](https://www.plugwise.com/en_US/products/anna).
+- For power monitoring there is one simply called the [P1](https://www.plugwise.com/en_US/products/smile-p1).
+- Although no longer sold, there also is support for Stretch, a gateway to create network connectivity for their older power products.
+
+Plugwise formerly sold power based products using a USB stick as the controller. This integration does not support the `Stick` directly, see [legacy power devices](#legacy-power-devices) for more information.
+
+## Platforms
+
+Depending on your specific Smile and connected devices, the following platforms will be available:
+
+- [Climate](#climate) for Adam and (a stand-alone) Anna.
+- [Binary Sensor](#binary_sensor) for status of your domestic hot water or secondary heater.
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)